Resources/config/mongodb.xml

```diff
@@ -5,25 +5,30 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="fos_oauth_server.client_manager.default" class="FOS\OAuthServerBundle\Document\ClientManager">
```

Comment:
Why?

Reply:
Yes, I know, but that's Symfony by default. If you're using Symfony 4, the services will be private by default.

My question is, why do you feel the services need to be public?

Reply:
Because they are managers, and they are therefore likely to be used elsewhere in the code. Personally, they are used in business services in my project.

Reply:
You can still use them, just just can't `get()` them (nor should you). The private->public part should be reverted.

Reply:
Indeed, in DI, it is not necessary to mark them in public, I change that.